examples/03_connectivity/plot_seed_based_correlation.py

```diff
@@ -169,8 +169,10 @@
 # interest.
 from nilearn import plotting
 
-display = plotting.plot_stat_map(seed_based_correlation_img, threshold=0.3,
-                                 cut_coords=pcc_coords[0])
+display = plotting.plot_stat_map(seed_based_correlations_fisher_z_img,
+                                 threshold=0.3, vmax=1.3,
```

Comment:
After a z-transform, thesholding at the level of .3 seems too liberal. 
vmax = 1.3 is also very low